README.md

```diff
@@ -13,7 +13,7 @@
 
 - [x] Support data backup and restore for PostgreSQL, MySQL and MongoDB
 - [x] Replace sensitive data with fake data
-- [x] Works on large database (> 10GB) (read [Design](#design))
+- [x] Works on large database (> 10GB) (read [Design](https://github.com/Qovery/replibyte/blob/main/docs/DESIGN.md))
```

Comment:
@sondrelg can you change with a relative link?

Reply:
Done 👍 Couldn't work out how to amend a commit in the Github UI so maybe just squash and merge if accepted 🙂